### DISCUSSION TITLE
Request limit - how does it work?

### DISCUSSION THREAD

**Discussion by GrizzlT** [HypixelDev/PublicAPI#350] [Q&A] [answered]
It is stated in the documentation that there is a limit of 120 req/min. Does that mean you are allowed to send 120 requests in a fixed timespan (every new minute for example) or are you allowed to only send 120 requests in the minute starting when you sent your first request?

I suspect the first one is applicable, if so how do we know when the server decides a minute has passed, how can we align our clocks?

**Comment by mdashlw** [comment, marked as answer]
You can use the `ratelimit-reset` header to know when the server decides to reset the limit for all keys.

**Reply by GrizzlT** [reply]
How would I use that header?

(EDIT: it's a response header for those who wonder like I did)

**Reply by lukematison** [reply]
> You can use the `ratelimit-reset` header to know when the server decides to reset the limit for all keys.

M dash
